src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -51,6 +53,10 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
     navigation.setParams({
       preassignedFareProduct: fp,
     });
+    if (fp.limitations.latestActivationDate && travelDate) {
+      if (isAfter(travelDate, formatISO(fp.limitations.latestActivationDate)))
```

Comment:
I would use `new Date` instead of `formatIso`, since in `isAfter` the iso is straightly parsed to a Date object. From performance measuring we see that handling date (like `parseIso`, `format`, etc.) is some of the cost that significantly adds up during add render.